api/envoy/admin/v3/config_dump.proto

```diff
@@ -348,29 +350,28 @@ message SecretsConfigDump {
   repeated DynamicSecret dynamic_warming_secrets = 3;
 }
 
-// [#not-implemented-hide:]
-// Envoy's EDS implementation *will* fill this message with all currently known endpoints. Endpoint
+// Envoy's admin fill this message with all currently known endpoints. Endpoint
 // configuration information can be used to recreate an Envoy configuration by populating all
 // endpoints as static endpoints or by returning them in an EDS response.
 message EndpointsConfigDump {
   message StaticEndpointConfig {
     // The endpoint config.
     google.protobuf.Any endpoint_config = 1;
 
-    // The timestamp when the Endpoint was last updated.
+    // [#not-implemented-hide:] The timestamp when the Endpoint was last updated.
```

Comment:
Yeah, I think you should add support for these fields (version, last update), by adding additional tracking state to cluster manager. I think it's fine to do that as a next PR though, the existing PR is a good size for review.

Reply:
What about [endpoint_stale_after](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/endpoint/v3/endpoint.proto#L101)? 
I think this value is used once and then dropped when EDS is initialized. Should I add additional tracking state to it?

Reply:
Interesting; I think the useful thing to track there would be how much time is remaining from when the config dump happens until the endpoint becomes stale. If it's really difficult to do, it's fine to leave as a TODO.

Reply:
Yes, I would like to add these support in next PR. For TTL, I'll try to add it, however, if it is difficult to achieve for now, maybe I'll leave it as a TODO and will try to solve it if there's still some time after I finish other milestones?